index.html

```diff
@@ -7,6 +7,8 @@
     <link rel="stylesheet" href="style.css">
     <script src="https://kit.fontawesome.com/4dc3d7e55b.js" crossorigin="anonymous"></script>
     <link href="https://fonts.googleapis.com/css2?family=Koulen&family=Luckiest+Guy&family=Poppins:wght@700&family=Special+Elite&family=Teko:wght@300&family=Yomogi&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;1,200&display=swap" rel="stylesheet">
     <title>Josué</title>
 </head>
 <body>
```

Comment:
**[OPTIONAL]** I highly recommend the tag usage for `header` like this 👇 

```
<header>
   <a>My Logo</a>
   <nav>
      <button>...</button>
  </nav>
</header>

```

This way it will be more readable and uses semantic tag correctly.